test/integration/contracts/mixins/with-events.spec.ts

```diff
@@ -117,6 +126,42 @@ describe('with-events mixin', () => {
 		);
 		assert(foo);
 
+		// Before enqueing up a create event we need to create a relationship to the random-slug type created above
```

Comment:
Same as other comment. Would it make more sense to update the types used in the test to match reality and remove the need to create a new relationship?

Reply:
In this case, the test creates a new plugin called `plugin-foobar`.
I think it would make more sense to update the plugin definition to include the needed relationships.
I'll do this after changing the plugin definitions to include relationships.